packages/config/config/devices/0x018c/pm-240-zw-3button.json

```diff
@@ -0,0 +1,25 @@
+{
+  "manufacturer": "Dawon",
+  "manufacturerId": "0x018c",
+  "label": "PM-S240-ZW",
```

Comment:
I think this is supposed to be
```suggestion
  "label": "PM-S340-ZW",
```
at least according to [this comment](https://github.com/zwave-js/node-zwave-js/issues/5132#issuecomment-1264405046) I apparently never followed up.

Can this be?

Reply:
If that's the case, I suppose we should also add the other product types mentioned in https://github.com/zwave-js/node-zwave-js/issues/5132#issuecomment-1264369553